src/web/frontend/redux/store/store.tsx

```diff
@@ -0,0 +1,15 @@
+import {createStore, applyMiddleware, compose} from 'redux'
+import combineReducer from '../reducer/combineReducer'
+import thunk from "redux-thunk";
+
+declare global {
```

Comment:
do not declare anything in `store` file
create a separate root-located file. something like `global.d.ts` and declare global and window interfaces there